pkg/newlog/cmd/newlogd.go

```diff
@@ -484,6 +509,11 @@ func handleDomainStatusImp(ctxArg interface{}, key string, statusArg interface{}
 		appD.msgIDAppCnt = d.msgIDAppCnt // inherit the counter for the app
 	}
 	domainUUID.Store(appD.appUUID, appD)
+
+	if status.DeploymentType == types.AppRuntimeTypeDocker {
+		runtimeAppUUID.Store(appD.appUUID, true)
```

Comment:
I think the value in the map can be of type `struct{}`, no need to use `bool` here

Reply:
going to remove this runtimeAppUUID.